client/gutenberg/editor/init.js

```diff
@@ -2,12 +2,13 @@
 /**
  * External dependencies
  */
-import { once } from 'lodash';
+import { flow, has, map, once, partialRight, partition } from 'lodash';
 
 /**
  * WordPress dependencies
  */
 import { use, plugins, dispatch } from '@wordpress/data';
+import { getBlockType, registerBlockType, unregisterBlockType } from '@wordpress/blocks';
```

Comment:
@vindl do we still have to avoid this (and use `require` instead?). I found no regressions by using this.

Reply:
What kind of issues there were before and how to spot them?

Reply:
It's fine now for `@wordpress/blocks` since it doesn't declare `@wordpresss/core-data` as a dependency. The problem was that `core-data` store is initialized statically whenever this package is imported. If that happens before the [data plugins](https://github.com/WordPress/gutenberg/tree/master/packages/data/src/plugins) are registered: https://github.com/Automattic/wp-calypso/blob/master/client/gutenberg/editor/init.js#L41-L44 we'll run into failing actions with cryptic console errors. Hence the `requires` to enforce proper sequencing.

Having said all that, `require` for `@wordpress/blocks` is not strictly necessary here, but I avoided using it intentionally before, just in case it (or some of its sub-deps) declares `core-data` as a dependancy in the future, which would resurface the old errors on npm package update. Could we please move this code outside of `init`, since it's not needed here anyway?

Reply:
To reproduce the above now, you can try:

`import { registerCoreBlocks } from '@wordpress/block-library' ;`

instead of requiring it bellow.